logistic-backend-python/services/logistics.py

```
@@ -29,6 +31,21 @@ def get_default_tags(mode: str) -> Dict[str, list]:
         raise ValueError(f"Неизвестный мод: {mode}")
+    
+def clear_cache_contents():
+    if os.path.exists("cache"):
+        for filename in os.listdir("cache"):
+            file_path = os.path.join("cache", filename)
+            try:
+                if os.path.isfile(file_path) or os.path.islink(file_path):
+                    os.unlink(file_path)
+                elif os.path.isdir(file_path):
+                    shutil.rmtree(file_path)
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)